tests/Util/FileHasherTest.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Core\Tests\Util;
+
+use Rector\Core\Kernel\RectorKernel;
+use Rector\Core\Util\FileHasher;
+use Rector\Testing\PHPUnit\AbstractTestCase;
+use Symplify\SmartFileSystem\SmartFileSystem;
+
+final class FileHasherTest extends AbstractTestCase {
+        private FileHasher $fileHasher;
+
+        private SmartFileSystem $smartFileSystem;
+
+        protected function setUp(): void
+        {
+            $this->boot();
+
+            $this->fileHasher = $this->getService(FileHasher::class);
+            $this->smartFileSystem = new SmartFileSystem();
+        }
+
+        public function testHash(): void
+        {
+            if (PHP_VERSION_ID < 80100) {
+                $this->markTestSkipped('xxh128 is not available on PHP 8.0');
+            }
+
+            $hash = $this->fileHasher->hash('some string');
+            $this->assertSame('8df638f91bacc826bf50c04efd7df1b1', $hash);
+        }
+
+        public function testHashFiles(): void
+        {
+            if (PHP_VERSION_ID < 80100) {
```

Comment:
This check is not needed, as test run on CI on php 8.1 already

Reply:
but people might run the test suite locally with a different php version?

Reply:
No, composer.json require php 8.1

https://github.com/rectorphp/rector-src/blob/0a7be603fcc467ac3956081bf83db9c0d5350eee/composer.json#L11

The phpunit itself not compatible with php 7.

It not testable in rector-src, except you want to add test on rector/rector that require php 7.2